autoconfigure/collector-kinesis/src/main/resources/zipkin-server-kinesis.yml

```diff
@@ -13,5 +13,9 @@ zipkin:
       aws-secret-access-key: ${KINESIS_AWS_SECRET_ACCESS_KEY:}
       # Optional AWS STS assume role ARN
       aws-sts-role-arn: ${KINESIS_AWS_STS_ROLE_ARN:}
-      # Optional AWS STS region endpoint to use
-      aws-sts-region: ${KINESIS_AWS_STS_REGION:us-east-1}
+      # Optional AWS STS region endpoint to use, defaults to KINESIS_AWS_REGION
+      aws-sts-region: ${KINESIS_AWS_STS_REGION:${zipkin.collector.kinesis.aws-kinesis-region}}
+      # Optional AWS Kinesis Region, defaults to AWS_REGION
+      aws-kinesis-region: ${KINESIS_AWS_REGION: ${zipkin.collector.kinesis.aws-region}}
```

Comment:
The extra space in the nested yaml is broke. I will raise a PR with a couple of other changes soon. 
caused by: org.yaml.snakeyaml.scanner.ScannerException: mapping values are not allowed here
 in 'reader', line 19, column 47:
     ... sis-region: ${KINESIS_AWS_REGION: ${zipkin.collector.kinesis.aws ...